### DISCUSSION TITLE
Firefox Background Updater for Mac

### DISCUSSION THREAD

**Discussion by Brownmattc** [mozilla/policy-templates#1177] [Q&A]
Hello,

We are deploying Firefox for MacOS from Jamf and controlling automatic updates through a configuration profile.

We notice that many machines have older versions installed on them. Users on these machines do not use Firefox.

Should we be expecting Firefox to update in the background on machines where the browser is not launched?

Thank you, Matt

**Comment by mkaply** [comment]
Firefox does not update in the background on macOS.
